### PR TITLE
Rename rollover crates

### PR DESCRIPTION
**Pull request by luckysori**
The `v_{1,2}_0_0` suffixes are noisy and too verbose. And using the word "deprecated" correctly flags the older version as one that will be phased out soon.

We leverage `pub use current::*` so that we don't have to add `current` as part of any imports.

### CODE REVIEW

**Review by Restioson — COMMENTED** [review]
If a version 3 of rollover happens, would support for v1 be removed?

**Comment by luckysori** [comment]
> If a version 3 of rollover happens, would support for v1 be removed?

Yep, that's why I think this is better: I think we are only ever going to support two versions of any given protocol at the same time, according to [ADR019](https://github.com/itchysats/architecture/blob/main/decision-records/019-maker-compatible-with-last-two-takers.md).

File: maker/src/actor_system.rs [hunk 1]
```diff
@@ -272,8 +268,8 @@ where
                 pong_address.clone(),
                 identify_listener_actor,
                 order,
-                rollover_v_1_0_0_addr.clone(),
-                rollover_v_2_0_0_addr.clone(),
+                rollover_deprecated_addr.clone(),
```
Comment (da-kami) [inline maker/src/actor_system.rs]: 🙃 maybe better to flip these so we always have `rollover_addr` first and the deprecated on second.

**Review by da-kami — APPROVED** [review]
I initially liked naming by version because it was explicit. But this is more intuitive when integrating the protocols, and the fact that we will unlikely support more than two version at the moment justifies this change. Let's do it :)

**Review by da-kami — APPROVED** [review]

**Pull request merged**